CHANGELOG.md

```diff
@@ -1,6 +1,80 @@
 ## 3.6.0 (TBC)
 
-Changes in progress for v3.6.0 are available at [.changes/v3.6.0](https://github.com/vmware/terraform-provider-vcd/tree/main/.changes/v3.6.0) until the release.
+## FEATURES
+* **New Data Source:** `vcd_org_group` allows to fetch an Organization Group to use it with other resources ([#798](https://github.com/vmware/terraform-provider-vcd/pull/798))
+* Add `catalog_version`, `number_of_vapp_templates`, `number_of_media`, `is_shared`, `is_published`, `publish_subscription_type` computed fields to catalog resource and datasource  ([#800](https://github.com/vmware/terraform-provider-vcd/pull/800))
+* Update `vcd_catalog` datasource so now it can take org from provider level or datasource level like modern resources/datasources ([#800](https://github.com/vmware/terraform-provider-vcd/pull/800))
+* **New Resource:** `vcd_nsxt_distributed_firewall` manages Distributed Firewall Rules in VDC Groups
+  ([#810](https://github.com/vmware/terraform-provider-vcd/pull/810))
```

Comment:
is this new line inside sentece is ok, as following lines? in other places it is one line and automatically wrapps

Reply:
Looked at github preview https://github.com/vmware/terraform-provider-vcd/blob/bafe71a5e7bc6ee1c9577806b0b76cc2e079edb4/CHANGELOG.md

I think it looks good:

<img width="1082" alt="image" src="https://user-images.githubusercontent.com/15804230/163335281-b02ad984-c09a-453b-87e2-a6721ae1fcba.png">